README.md

```diff
@@ -213,9 +213,38 @@ To view the STDOUT from a docker container of a running server, you can invoke
 
 To stop everything, use `docker-compose down`.
 
+To run the ruby console hooked into the current rails session, run
+
+`docker-compose exec backend rails c`
+
+To manually inspect the binary snapshots in  the `pg_dump` folder, run
+
+`pg_restore  2020_04_26__01_12_39_tapp_development.psql`
+
+which will display the output to the command line (it won't actually insert into a database.)
+
+##### Running tests from a ram-drive
+
+Since the test suite clears the database several times, it can be slow! To speed things
+up by a factor of ~10x, you can run the database from a ram drive instead. In linux, you can
+set up your ram drive with
+
+```
+cd <tapp home director>
+mkdir /dev/shm/pg_data
```

Comment:
I get an error when running this.
`mkdir: /dev/shm: No such file or directory`

Reply:
Are you on linux? Or are you on mac? Whatever the case, figure out how to get a ram drive set up and put instructions that work into the readme.